services/velero/4.1.2/helmrelease/helmrelease.yaml

```diff
@@ -11,7 +11,7 @@ spec:
         kind: HelmRepository
         name: vmware-tanzu.github.io
         namespace: kommander-flux
-      version: 4.1.1
+      version: 4.1.2
```

Comment:
didn't technically need to bump the helm chart version but I guess since this version does exist doesn't hurt 🤷🏻‍♀️